packages/context/src/interceptor.ts

```diff
@@ -390,7 +390,7 @@ export function invokeMethodWithInterceptors(
       const interceptors = invocationCtx.loadInterceptors();
       return invokeInterceptors(invocationCtx, interceptors);
     },
-    () => invocationCtx.close(),
+    () => invocationCtx.clearSubscriptions(),
```

Comment:
Please add a test that fails before this change is made and passes with the fix in place.

Ideally, I'd like to see acceptance-level test demonstrating a somewhat realistic scenario that is broken now.